tracer/src/Datadog.Trace/Iast/Aspects/System.Security.Cryptography/HashAlgorithmAspect.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="HashAlgorithmAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using Datadog.Trace.ClrProfiler;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.HashAlgorithm;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Iast.Aspects
+{
+    /// <summary> HashAlgorithm class aspects </summary>
+    [AspectClass("mscorlib,System.Security.Cryptography.Primitives,System.Security.Cryptography")]
```

Comment:
Do we actually want to use callsite instrumentation in non-.NET Framework 🤔 It seems like it would still be preferable for performance reasons to use CallTarget instrumentation in `System.Security.Cryptography.Primitives` and `System.Security.Cryptography`, and only use this for `mscorlib`?

Reply:
Yes, that was an option. I have been thinking about it and although it is cleaner having the same instrumentation system, for performance reasons I have no problem going back to the old instrumentation and use callsite only for net462

Reply:
Done!